mesonbuild/dependencies/base.py

```diff
@@ -1801,6 +1801,11 @@ def _shebang_to_cmd(script):
                     # Windows does not ship python3.exe, but we know the path to it
                     if len(commands) > 0 and commands[0] == 'python3':
                         commands = mesonlib.python_command + commands[1:]
+                elif mesonlib.is_cygwin():
+                     for i in commands:
+                         if 'python3' in i:
```

Comment:
__[Flake8]__ [E111] indentation is not a multiple of four

<sub>posted by Sider</sub>